modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -325,4 +332,117 @@ export default class NestedTabs extends Base {
 			}
 		}
 	}
+
+	getActiveClass() {
+		const settings = this.getSettings();
+
+		return settings.classes.active;
+	}
+
+	getVisibleTabTitle( tabTitleFilter ) {
+		const $tabTitle = this.elements.$tabTitles.filter( tabTitleFilter ),
+			isTabTitleDesktopVisible = null !== $tabTitle[ 0 ]?.offsetParent,
+			$visibleTabTitle = isTabTitleDesktopVisible ? $tabTitle[ 0 ] : $tabTitle[ 1 ];
+
+		return $visibleTabTitle;
+	}
+
+	getKeyPressed( event ) {
+		const keyTab = 9,
+			keyEscape = 27,
+			isTabPressed = keyTab === event?.which,
+			isShiftPressed = event?.shiftKey,
+			isShiftAndTabPressed = !! isTabPressed && isShiftPressed,
+			isOnlyTabPressed = !! isTabPressed && ! isShiftPressed,
+			isEscapePressed = keyEscape === event?.which;
+
+		if ( isShiftAndTabPressed ) {
+			return 'ShiftTab';
+		} else if ( isOnlyTabPressed ) {
+			return 'Tab';
+		} else if ( isEscapePressed ) {
+			return 'Escape';
+		}
+	}
+
+	changeFocusFromContentContainerItemBackToTabTitle( event ) {
+		if ( this.hasDropdownLayout() ) {
+			return;
+		}
+
+		const isShiftAndTabPressed = 'ShiftTab' === this.getKeyPressed( event ),
+			isOnlyTabPressed = 'Tab' === this.getKeyPressed( event ),
+			isEscapePressed = 'Escape' === this.getKeyPressed( event ),
+			firstItemIsInFocus = this.itemInsideContentContainerHasFocus( 0 ),
+			lastItemIsInFocus = this.itemInsideContentContainerHasFocus( 'last' ),
+			activeTabTitleFilter = `.${ this.getActiveClass() }`,
+			$activeTabTitleVisible = this.getVisibleTabTitle( activeTabTitleFilter ),
+			activeTabTitleIndex = parseInt( $activeTabTitleVisible?.getAttribute( 'data-tab' ) ),
+			nextTabTitleFilter = this.getTabTitleFilterSelector( activeTabTitleIndex + 1 ),
+			$nextTabTitleVisible = this.getVisibleTabTitle( nextTabTitleFilter ),
+			pressShiftTabOnFirstFocusableItem = isShiftAndTabPressed && firstItemIsInFocus && !! $activeTabTitleVisible,
+			pressTabOnLastFocusableItem = isOnlyTabPressed && lastItemIsInFocus && !! $nextTabTitleVisible;
+
+		if ( ( pressShiftTabOnFirstFocusableItem ) || isEscapePressed ) {
+			event.preventDefault();
+
+			$activeTabTitleVisible?.focus();
+		} else if ( pressTabOnLastFocusableItem ) {
+			event.preventDefault();
+
+			this.setTabindexOfActiveContainerItems( '-1' );
+
+			$nextTabTitleVisible?.focus();
+		}
+	}
+
+	changeFocusFromActiveTabTitleToContentContainer( event ) {
+		const isOnlyTabPressed = 'Tab' === this.getKeyPressed( event ),
+			$focusableItems = this.getFocusableItemsInsideActiveContentContainer(),
+			$firstFocusableItem = $focusableItems[ 0 ],
+			$currentTabTitle = elementorFrontend.elements.window.document.activeElement,
+			currentTabTitleIndex = parseInt( $currentTabTitle?.getAttribute( 'data-tab' ) );
```

Comment:
Is there a situation in which `currentTabTitle` will be undefined?
Why are you using optional chaining?

Reply:
Removed.